app/scripts/lib/origin-check.js

```diff
@@ -25,11 +25,14 @@
 
 define([
   'lib/promise',
-  'lib/channels/iframe'
-], function (p, IFrameChannel) {
+  'lib/channels/iframe',
+  'raven'
+], function (p, IFrameChannel, Raven) {
   'use strict';
 
-  var RESPONSE_TIMEOUT_MS = 100;
+  // This timeout needs to be long enough for slow machines to able to respond to
+  // See https://github.com/mozilla/fxa-content-server/issues/2946 for details
+  var RESPONSE_TIMEOUT_MS = 5000;
```

Comment:
Is there a reason for 5000? or is it a magic number? from irc: `absolute worst case was 1634ms`.